firestore-bigquery-export/functions/src/events.ts

```diff
@@ -2,14 +2,39 @@ import * as eventArc from "firebase-admin/eventarc";
 
 const { getEventarc } = eventArc;
 
-const EXTENSION_NAME = "firestore-bigquery-export";
+/**
+ * The extension name is used to construct event types dynamically.
+ * Changing this name affects the new event type generation.
+ * @constant EXTENSION_NAME
+ */
+const EXTENSION_NAME =
+  process.env.EXT_INSTANCE_ID || "firestore-bigquery-export";
 
-const getEventType = (eventName: string) =>
-  `firebase.extensions.${EXTENSION_NAME}.v1.${eventName}`;
+/**
+ * Generates both the OLD and NEW event types to maintain backward compatibility.
+ *
+ * Old Event Type: firebase.extensions.firestore-counter.v1.{eventName}
+ * New Event Type: firebase.extensions.firestore-bigquery-export.v1.{eventName}
+ *
+ * @param eventName The name of the event (e.g., "onStart", "onError", etc.)
+ * @returns An array containing both the old and new event types
+ */
+const getEventTypes = (eventName: string) => [
+  `firebase.extensions.firestore-counter.v1.${eventName}`, // OLD Event Type for backward compatibility
+  `firebase.extensions.${EXTENSION_NAME}.v1.${eventName}`, // NEW Event Type following the updated convention
```

Comment:
```suggestion
  `firebase.extensions.firestore-bigquery-export.v1.${eventName}`, // NEW Event Type following the updated convention
```

Reply:
Same as before: I added this based on your previous comments regarding several instances of the same extension.